network/src/lib.rs

```diff
@@ -114,6 +124,10 @@ where
 			});
 		}
 
+		if (self.is_major_syncing)() {
+			return Ok(Validation::Success { is_new_best: true });
+		}
```

Comment:
```suggestion
		if (self.is_major_syncing)() {
			return Ok(Validation::Success { is_new_best: false });
		}
```

And please move it above the other if and remove the new added inner check.

Reply:
done